smoke_tests/run_smoke_tests.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -e
+
+DOCKER_IMAGE=rig:smoke_test
+COMPOSE="docker-compose -f smoke_tests.docker-compose.yml"
+
+function section_header() {
+  echo ""
+  echo "╓─────[ $@ ]─────╖"
+  echo ""
+}
+
+section_header "Building docker image: ${DOCKER_IMAGE}"
+docker build -f ../smoke_tests.dockerfile -t "${DOCKER_IMAGE}" ..
+
+section_header "Starting Kafka and backend-simulation services"
+$COMPOSE up -d kafka rest-api
+
+section_header "Running tests, plaintext Kafka connection"
```

Comment:
Thumbs up for testing both cases!

Reply:
_Ideally_ we would not rerun _all_ the tests, but yeah, should be alright for now